### PR TITLE
add 1.24 release notes

### PR DESCRIPTION
**Pull request by codyjlandstrom**

### CODE REVIEW

File: src/content/release-notes.mdx [hunk 1]
```diff
@@ -5,6 +5,40 @@ sidebar_label: Release notes
...
+
+- Add provider credentials query for deploy and test executions  <!-- https://github.com/okteto/app/pull/8454 -->
+- Enable [multi-user token authentication](self-hosted/install/auth/token.mdx)  <!-- https://github.com/okteto/app/pull/8481 -->
+- Announcing Okteto CLI 3.0 [Will link to blog post]
+
+### Improvements
+
+- Added new Developer Quickstart Guide to the documentation <!-- to add link on version upgrade -->
+- Updated the "Want to know more" links in an empty namespace  <!-- 8415 -->
+- Added tolerations for EventsExporter StatefulSet  <!-- 8446 -->
```
Comment (pchico83) [inline src/content/release-notes.mdx:28]: This is a feature, not an improvement. @jvc5546 Could you document the event exporter section in our helm values?

File: src/content/release-notes.mdx [hunk 2]
```diff
@@ -5,6 +5,40 @@ sidebar_label: Release notes
...
+- Updated pipeline-runner image to v1.0.5 to update cue to 0.9.2 to fix vulnerabilities  <!-- 8482 -->
```
Comment (pchico83) [inline src/content/release-notes.mdx:29]: We never mention vulnerabilities, unless it's a patch release

File: src/content/release-notes.mdx [hunk 3]
```diff
@@ -5,6 +5,40 @@ sidebar_label: Release notes
...
+
+9 September 2024
+
+This versions is compatible with Kubernetes versions 1.27 to 1.30
+
+### Breaking Changes
+
+- Upgraded ingress-nginx to v4.11.2  <!-- 8460 -->
+
+### New Features
```
Comment (pchico83) [inline src/content/release-notes.mdx:18]: @codyjlandstrom we are adding support for buildkit bridge mode. @jLopezbarb could you create a separate PR to document it in the helm values? And explain the advantages of using bridge mode?

File: src/content/release-notes.mdx [hunk 4]
```diff
@@ -5,6 +5,40 @@ sidebar_label: Release notes
...
+
+- Add provider credentials query for deploy and test executions  <!-- https://github.com/okteto/app/pull/8454 -->
+- Enable [multi-user token authentication](self-hosted/install/auth/token.mdx)  <!-- https://github.com/okteto/app/pull/8481 -->
+- Announcing Okteto CLI 3.0 [Will link to blog post]
+
+### Improvements
```
Comment (pchico83) [inline src/content/release-notes.mdx:24]: We always mention the upgrade to the Okteto CLI version. In this case, we are upgrading to CLI 2.31.0.
The main features are:
- Replace RSA by ECDSA for SSH keys. This allows the use of debuggers with Visual Studio with Development Containers
- Prevent syncthing to restart and activating [relay](https://docs.syncthing.net/users/relaying.html) if the synching configuration is lost

File: src/content/release-notes.mdx [hunk 5]
```diff
@@ -5,6 +5,40 @@ sidebar_label: Release notes
...
 
+## 1.24
+
+9 September 2024
+
+This versions is compatible with Kubernetes versions 1.27 to 1.30
+
+### Breaking Changes
+
+- Upgraded ingress-nginx to v4.11.2  <!-- 8460 -->
```
Comment (pchico83) [inline src/content/release-notes.mdx:16]: Let's add a link to the nginx release notes:
https://github.com/kubernetes/ingress-nginx/releases/tag/controller-v1.11.2

**Review by pchico83 — CHANGES_REQUESTED** [review]

File: src/content/release-notes.mdx (hunk 3 above)
Comment (pchico83) [reply src/content/release-notes.mdx:18]: PR documenting buildkit network mode:
https://github.com/okteto/docs/pull/819

File: src/content/release-notes.mdx [hunk 6]
```diff
@@ -5,6 +5,43 @@ sidebar_label: Release notes
...
 
+## 1.24
+
+9 September 2024
+
+This versions is compatible with Kubernetes versions 1.27 to 1.30
+
+### Breaking Changes
+
+- Upgraded [ingress-nginx to v1.11.2](https://github.com/kubernetes/ingress-nginx/releases/tag/controller-v1.11.2)  <!-- 8460 -->
```
Comment (pchico83) [inline src/content/release-notes.mdx:16]: @codyjlandstrom the breaking change is a very uncommon case. I would explain more about it. Maybe something like "This `ingress-nginx` release comes with a breaking change only affecting edge case" And link to the issue where it's explained:
https://github.com/kubernetes/ingress-nginx/issues/11866
They didn't document it as a breaking change on their release notes

**Review by pchico83 — APPROVED** [review]

File: src/content/release-notes.mdx [hunk 7]
```diff
@@ -5,6 +5,40 @@ sidebar_label: Release notes
...
+This versions is compatible with Kubernetes versions 1.27 to 1.30
+
+### Breaking Changes
+
+- Upgraded ingress-nginx to v4.11.2  <!-- 8460 -->
+
+### New Features
+
+- Add provider credentials query for deploy and test executions  <!-- https://github.com/okteto/app/pull/8454 -->
+- Enable [multi-user token authentication](self-hosted/install/auth/token.mdx)  <!-- https://github.com/okteto/app/pull/8481 -->
```
[the remark this answers is not in the document]
Comment (codyjlandstrom) [reply src/content/release-notes.mdx:21]: https://github.com/okteto/docs/pull/820

File: src/content/release-notes.mdx (hunk 1 above)
Comment (pchico83) [reply src/content/release-notes.mdx:28]: @codyjlandstrom added a PR to document the event exporter. I would link to the event exporter docs here:
https://github.com/okteto/docs/pull/821

**Pull request merged**